cli/src/cli_args.rs

```diff
@@ -137,11 +137,7 @@ pub enum Command {
     /// Resets the checksums records what is used to check if there were any code changes for
     /// each target.
     #[clap(visible_alias = "reset")]
-    ResetChecksum {
-        /// Reset release records
-        #[arg(short, long, default_value_t = false)]
-        production: bool,
-    },
+    ResetChecksum,
```

Comment:
Maybe only naming it "reset", as user would not care about details (which could be in help description, only).

Reply:
I think `reset` only could be a little bit ambiguous and could mean `clean` as well. 
Actually, this command can be used with `reset` only because it has as an alias :grinning: 

```
reset-checksum    Resets the checksums records what is used to check if there were any code changes for each target [aliases: reset]
```